modules/hooks.nix

```diff
@@ -3414,6 +3414,13 @@ lib.escapeShellArgs (lib.concatMap (ext: [ "--ghc-opt" "-X${ext}" ]) hooks.ormol
         entry = "${hooks.typstfmt.package}/bin/typstfmt";
         files = "\\.typ$";
       };
+      typstyle = {
+        name = "typstyle";
+        description = "Beautiful and reliable typst code formatter";
+        packages = tools.typstyle;
+        entry = "${hooks.typstyle.package}/bin/typstyle";
```

Comment:
You can check whether `tools.typstyle` is `null` and print a nicer error message if it is. Search for `lib.throwIf` for examples.